src/css/cssMode.ts

```diff
@@ -95,7 +95,7 @@ export function setupMode(defaults: LanguageServiceDefaults): IDisposable {
 		}
 		if (modeConfiguration.diagnostics) {
 			providers.push(
-				new languageFeatures.DiagnosticsAdapter(languageId, worker, defaults.onDidChange)
+				new languageFeatures.DiagnosticsAdapter(languageId, worker, defaults.onDidChange, defaults.modeConfiguration?.diagnosticCodesToIgnore)
```

Comment:
You don't need this feature in CSS, right? If true, consider to undo these changes and open a PR to the official repo with both changes.